player/lua/stats.lua

```diff
@@ -35,7 +35,7 @@ local o = {
     file_tag_max_length = 128,       -- only show file tags shorter than this length in bytes
     file_tag_max_count = 16,         -- only show the first x file tags
     show_frame_info = false,         -- whether to show the current frame info
-    term_width_limit = -1,           -- overwrites the terminal width
+    term_clip = true,
     term_height_limit = -1,          -- overwrites the terminal height
```

Comment:
Is that useful? We probably should remove this for parity with width. I don't see a reason to make it so much configurable. Even console doesn't have things like that.

Reply:
Removed.